docs/DeveloperGuide.md

```diff
@@ -956,38 +956,60 @@ testers are expected to do more *exploratory* testing.
-   
-### Storage
-1. With the addition of the `Appointment` class, we had to learn how the JSON format worked, and how we could utilise its functionalities to allow for the successful storing and reading of data, now that every `Person` has an `Appointment` field.
-2. It was necessary to get a basic understanding of how JSON files work. Upon creating our new Json class that adapts to appointment, a weak understanding of `@JsonCreator` property caused initial bugs where appointment information had issues being retrieved from the Json file format. This bug was eventually fixed with greater understanding of Json.
+**Addition of New Fields to Patients**  
+New fields such as diagnosis, medications, notes and appointments were added to align WardWatch with the requirements of a healthcare management application.
 
-### UI
-1. UI Design required good understanding and identification of the relevant dependencies on the corresponding .fxml and .css files.
-2. To restructure and customise the UI, a firm understanding of UI elements such as the HBox, VBox and Stackpanes is necessary to layout the UI elements according. There was particularly a struggle understanding the effects of the growing Hbox and Vbox, as the space occupied is unexpected.
-3. Choice or colour to make the app aesthetic is also non-trivial as it is difficult to find matching colours that are aesthetic.
-4. The setting of result boxes also required consideration, as too long or short may cause data to not be represented clearly, due to the text overflowing. It is also a tradeoff for space with the other elements.
+**Find Command Enhancements**  
+Our team improved on the existing find command of AB3 to allow for greater flexibility. Users can now search for patients not only by name but also by other fields like diagnosis and medication. The enhanced command also enables users to filter patients based on specific criteria, which required us to redesign and test the search functionality thoroughly.
 
+**Patient Tagging**  
+We modified the original tag feature to automatically tag contacts in the WardWatch system. Patients with a unique ID are tagged as patients to streamline the management of patient data and make it easier for healthcare providers to organize contacts.
+
+**User Interface (UI) Improvements**  
+To enhance the user experience, we redesigned the interface to accommodate WardWatch’s new features and match the healthcare theme. Additionally, we adapted the UI logic for easier modification by future developers.
+
+### New Features
+
+**Copy Command**  
+Our team introduced a new copy command that allows users to copy contact information, such as patient email addresses, for mass communication purposes.
```

Comment:
May I ask which feature is this part referring to?

Reply:
oh shit i forgot to remove that ps AAHAHHA